### PR TITLE
Add info about app importance sensor

### PR DESCRIPTION
**Pull request by dshokouhi**
Docs for: https://github.com/home-assistant/android/pull/1289

### CODE REVIEW

File: docs/core/sensors.md [hunk 1]
```diff
@@ -72,6 +72,7 @@ All sensors update during a periodic 15-minute interval and they will also updat
 | `binary_sensor.power_save` | None | Whether or not the device is in power saving mode. |
 | `sensor.activity` | `confidence` | The current activity type as computed by Google. Requires activity recognition permissions on supported devices. |
 | [App Data Sensors](#app-data_sensors) | None | Sensors that show how much data was sent or received by the app. |
+| `sensor.app_importance` | None | The current importance of the app to determine if its in the foreground or cached. |
```
Comment (zacwest) [inline docs/core/sensors.md:75]: link to the section below from here?

Comment (dshokouhi) [reply docs/core/sensors.md]: good call :)

**Review by zacwest — APPROVED** [review]
(i'll let you merge when you feel comfortable)

**Comment by dshokouhi** [comment]
Thanks! 😃 We can merge this after the parent PR has been merged

**Pull request merged**